pterodactyl/pterodactyl.php

```diff
@@ -308,7 +315,12 @@ function pterodactyl_CreateAccount(array $params) {
             elseif(isset($envName)) $environment[$var] = $envName;
             else $environment[$var] = $default;
         }
-
+        if (empty(pterodactyl_GetOption($params, 'server_name'))){
```

Comment:
3rd arg of pterodactyl_GetOption supports default option, just do:
`$name = pterodactyl_GetOption($params, 'server_name', pterodactyl_GenerateUsername() . '_' . $params['serviceid']);`